plugins/outputs/mqtt/mqtt.go

```diff
@@ -5,15 +5,14 @@ import (
 	// Blank import to support go:embed compile directive
 	_ "embed"
 	"fmt"
-	"net/url"
-	"strings"
-	"sync"
-
 	"github.com/influxdata/telegraf"
 	"github.com/influxdata/telegraf/config"
 	"github.com/influxdata/telegraf/plugins/common/tls"
 	"github.com/influxdata/telegraf/plugins/outputs"
 	"github.com/influxdata/telegraf/plugins/serializers"
+	"net/url"
+	"strings"
+	"sync"
```

Comment:
Please keep those in the old place. We usually have 3 groups of imports (separated by an empty line): built-in Golang libraries, 3rd-party libraries and telegraf internal libraries (everything starting with `github.com/influxdata/telegraf`. Each group should be sorted alphabetically...